docs/changelog/2021.md

```diff
@@ -0,0 +1 @@
+- Speed up the creation of the Bspline interpolation, by switching the matrix A inside the Bspline class to a sparse matrix
```

Comment:
Could you please reword this to start with `- Improved `?
This results in a more digestible changelog.